services/files.js

```diff
@@ -4,13 +4,12 @@ const parseWikiUrl = require('./util/parseWikiUrl');
 const errors = require('./util/errors');
 
 async function getImageTitles({ client, title, wikiUrl }) {
-  const response = await client.getResultsFromApi([title], 'images', wikiUrl);
-  assert.ok(response.pages, errors.emptyResponse);
-  const pages = Object.values(response.pages);
-  assert.ok(pages.length === 1);
-  const { images = [] } = pages[0];
-
-  return images.map(image => image.title);
+  const params = { imlimit: 500 };
+  const { pages } = await client.getResultsFromApi([title], 'images', wikiUrl, params);
+  return Object.values(pages).reduce((acc, page) => {
+    const { images = [] } = page;
+    return [...acc, ...images.map(image => image.title)];
+  }, []);
```

Comment:
You could extract this reduction logic into a separate function and give it a speaking name like `mergePages(pages)` to make it more descriptive.

Reply:
I think™ I did this while you were writing this 😹 

Basically using a "formatter" function, like I do in the other services.